store/populations.go

```diff
@@ -564,6 +564,8 @@ func (s *store) GetStats(ctx context.Context, in *pb.GetStatsRequest,
 			continue
 		} else if t.Predicate == "censusACSTableId" {
 			continue
+		} else if t.Predicate == "constrainingProperties" {
```

Comment:
This should be "constraintProperties". The bug says "constrainingProperties", but I re-used the existing property  "constraintProperties".

Reply:
Okay, I updated the bug title as well :)